nengo/utils/tests/test_cache.py

```diff
@@ -11,6 +11,8 @@ def test_human2bytes():
     assert human2bytes('1 MB') == 1048576
     assert human2bytes('1.5 GB') == 1610612736
     assert human2bytes('14 B') == 14
+    assert human2bytes('1.5GB') == 1610612736
```

Comment:
I believe this case was already working. There should be a test like
`assert human2bytes('1B') == 1`